src/analysis/normed_space/multilinear.lean

```diff
@@ -728,7 +728,7 @@ variables (𝕜 n) (A : Type*) [normed_ring A] [normed_algebra 𝕜 A]
 `m` the product of all the `m i`.
 
 See also: `multilinear_map.mk_pi_algebra`. -/
-protected def mk_pi_algebra_fin : continuous_multilinear_map 𝕜 (λ i : fin n, A) A :=
```

Comment:
This will conflict with #13426.

Reply:
I'll revert this line. Could you please use notation in #13426?

Reply:
Thanks!

Reply:
I hope, this PR no longer conflicts with yours.